Firmware/mmu2_protocol_logic.cpp

```diff
@@ -495,7 +509,7 @@ StepStatus ProtocolLogic::IdleStep() {
     return Finished;
 }
 
-ProtocolLogic::ProtocolLogic(MMU2Serial *uart, uint8_t extraLoadDistance)
+ProtocolLogic::ProtocolLogic(MMU2Serial *uart, uint8_t extraLoadDistance, uint8_t pulleySlowFeedrate)
```

Comment:
Just a thought. If we add more registers here, adding a new parameter each time isn't very scalable. We should consider  creating a struct of some kind to reduce the parameters 🤔

Reply:
I agree, for now it is good enough, in the future we can introduce some data structure